constructr-coordination/src/main/scala/de/heikoseeberger/constructr/coordination/ConsulCoordination.scala

```diff
@@ -113,8 +113,7 @@ final class ConsulCoordination(prefix: String, clusterName: String, host: String
     }
     val createSessionUri = sessionUri
       .withPath(sessionUri.path / "create")
-      .withQuery(Uri.Query("Behaviour" -> "delete", "ttl" -> toSeconds(ttl)))
-    send(Put(createSessionUri)).flatMap {
+    send(Put(createSessionUri, s"""{"behavior": "delete", "ttl": "${toSeconds(ttl)}s"}""")).flatMap {
```

Comment:
We've added the media type for clarity and good practice. BTW, do you know why this branch has conflicts with master?.